plugins/scorecards/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@backstage/plugin-scorecards",
+  "version": "0.1.0",
```

Comment:
now that we are meant to make changesets even for new packages (see the comment for that elsewhere), you can leave this as `0.0.0`